doc/sphinx/source/recipes/recipe_climwip.rst

```diff
@@ -114,7 +145,7 @@ multiple datasets.
 References
 ----------
 
-* `Brunner et al. 2020 (accepted), Earth Syst. Dynam., <https://doi.org/10.5194/esd-2020-23>`_.
+* `Brunner et al. 2020, Earth Syst. Dynam., <https://doi.org/10.5194/esd-11-995-2020>`_.
```

Comment:
maybe add Lorenz et al 2018, Knutti et al 2017 to references? they are in references in the scripts, and even if method changed a bit in meantime, the main method is still the same.
and Brunner et al 2020 needs updating with volume and pages.